src/main/flight/navigation.c

```diff
@@ -113,7 +113,7 @@ static void GPS_calc_longitude_scaling(int32_t lat);
 static void GPS_calc_velocity(void);
 static void GPS_calc_location_error(int32_t * target_lat, int32_t * target_lng, int32_t * gps_lat, int32_t * gps_lng);
 
-#ifdef USE_NAV
+#ifdef NAV
```

Comment:
Some of your changes are reverting another recent merge:  https://github.com/betaflight/betaflight/commit/c0ce62d601cbb18134092416eade842dea848c99#diff-3be4a0626565051981a50eb12f37aee7

Not sure which is correct - just remembered seeing the USE_NAV from an earlier commit.

Reply:
3.3 has been switched to consistently use the `USE_` prefix for defines that enable features. 3.2 has not - and this pull request is for 3.2.